### ISSUE TITLE
Tracker sensibility

### ISSUE THREAD

**Issue by jeremypele** [danielgek/nativescript-sentry#6]
Good job on the last updates, really cool!

Got a bit frustrated on development mode on the other hand as every issue is captured without output on the console..Is there any way to configure the 'sensibility' of the tracker?

**Comment by danielgek** [comment]
@jeremypele yes, i got stuck at one thing when implementing the options, will add it in the next version ;)

And thank you :D

**Comment by jeremypele** [comment]
Ok cool. Putting the upgrade on a side branch then 👍

**Comment by danielgek** [comment]
I will ping you when it's done, probably in the next few days

**Comment by danielgek** [comment]
@jeremypele in lastest version i have disabled it so now the logs are cleaner, will add options later

**Comment by jeremypele** [comment]
Thanks I'll try that ASAP

Cheers!
